static/app/views/settings/account/accountDetails.tsx

```diff
@@ -24,10 +58,10 @@ class AccountDetails extends DeprecatedAsyncView<Props> {
     return [['user', ENDPOINT]];
   }
 
-  handleSubmitSuccess = (user: User) => {
+  handleSubmitSuccess = (user: ChangeAvatarUser) => {
     // the updateUser method updates our Config Store
     // No components listen to the ConfigStore, they just access it directly
-    updateUser(user);
+    updateUser(user as User);
```

Comment:
Casting this as `User` means we are throwing away the `ChangeAvatarUser` type. This means in that `updateUser`, typescript will incorrectly allow us to access `user.options.theme` because `User` has `options` as always defined.